error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/AssertJEnumerableRules.java

```diff
@@ -47,6 +47,13 @@ AbstractIntegerAssert<?> before(AbstractIterableAssert<?, ?, E, ?> enumAssert) {
       return Refaster.anyOf(enumAssert.size().isNotEqualTo(0), enumAssert.size().isPositive());
     }
 
+    @BeforeTemplate
+    AbstractIterableAssert<?, ?, E, ?> before2(AbstractIterableAssert<?, ?, E, ?> enumAssert) {
+      return Refaster.anyOf(
+          enumAssert.size().isNotEqualTo(0).returnToIterable(),
+          enumAssert.size().isPositive().returnToIterable());
+    }
```

Comment:
Arguably the newly added templates are the correct ones, while the existing ones aren't fully correct. Will add a commit to swap them and call this out.